cypress/e2e/trigger_builder.cy.js

```diff
@@ -23,8 +23,8 @@ const triggerOperatorToLabel = {
   not_contains: 'Not Contains',
 };
 
-const setupTriggerEditorFromSource = (trigger) => {
-  cy.get('#triggerSource').scrollIntoView().paste(JSON.stringify(trigger));
+const setupTriggerEditorFromSource = (json_object) => {
+  cy.pasteJsonIntoEditor({ json_object });
```

Comment:
The input param is still a trigger
```suggestion
const setupTriggerEditorFromSource = (trigger) => {
  cy.pasteJsonIntoEditor({ json_object: trigger });
````